addons/common/functions/fnc_getItemReplacements.sqf

```diff
@@ -8,7 +8,7 @@
  * 1: Include type replacements for items <BOOL> (default: true)
  *
  * Return Value:
- * Item replacements <ARRAY of STRINGs> if replacements exist, nil if not
+ * Item replacements <ARRAY of STRINGs> if replacements exist, <NIL> if not
```

Comment:
```suggestion
 * Item replacements (strings if replacements exist, nil if not) <ARRAY of STRINGs or NIL>
```

I'm not a big fan of doing it like this, but I believe it's necessary for the parsing of public functions for the wiki.
https://ace3.acemod.org/wiki/functions/common#ace_common_fnc_getitemreplacements

As it stands there's nothing mentioning nil for this specific function, so in general we might have to move these small descriptions before the item types in the header descriptions.